clients/spark/core/src/main/scala/io/treeverse/clients/KeyFilter.scala

```diff
@@ -0,0 +1,12 @@
+package io.treeverse.clients
+
+trait KeyFilter extends Serializable {
+  // Number of rounds for this matcher.
+  // Matcher logic could be changed based on the number of rounds.
```

Comment:
```suggestion
  // Rounds are counted from 1, allowing matcher logic could be changed based on the number of rounds.
```
(the important bit is to count from 1)